src/appengine/handlers/jobs.py

```diff
@@ -44,6 +57,37 @@ def get_queues():
   return queues
 
 
+def get_results(this):
+  """Get results for the jobs page."""
+  params = {k: v for k, v in this.request.iterparams()}  # pylint: disable=unnecessary-comprehension
```

Comment:
Let's just write this as `dict(this.request.iterparams())` to avoid the lint error.

Reply:
thanks! Will replace this